src/views/Main.js

```diff
@@ -0,0 +1,83 @@
+import './Main.css';
+import { useEffect, useState } from 'react';
+import { fetchFilteredPokemon, fetchPokemon, fetchTypes } from '../services/pokemon';
+import PokeCard from '../components/Pokecard';
+import TypeSelector from '../components/TypeSelector';
+import Search from '../components/Search';
+
+export default function Main() {
+  const [pokemon, setPokemon] = useState([]);
+  const [types, setTypes] = useState([]);
+  const [selectedType, setSelectedType] = useState('');
+  const [searchPokemon, setSearchPokemon] = useState('');
+  const [sorted, setSort] = useState('asc');
+  const [page, setPage] = useState(1);
+  const [loading, setLoading] = useState(true);
+
+  const nextPage = () => {
+    setPage(page + 1);
+  };
+
+  const prevPage = () => {
+    setPage(page - 1);
+    if (page <= 1) {
+      setPage(1);
+    }
```

Comment:
Awesome work adding pagination. If you wanted to prevent the user from going over the page limit, it would be good to get the length of data back and set the max page limit to the length of Math.ceil(data / 10), so that it is always rounded up by one page.